allensdk/brain_observatory/behavior/session_apis/data_transforms/behavior_ophys_data_transforms.py

```diff
@@ -275,15 +275,23 @@ def get_sync_licks(self):
         return pd.DataFrame({'time': lick_times})
 
     @memoize
-    def get_licks(self):
+    def get_licks(self) -> pd.DataFrame:
+        """
+        Returns
+        -------
+        pd.DataFrame
+            Two columns: "time", which contains the sync time
+            of the licks that occurred in this session and "frame",
+            the frame numbers of licks that occurred in this session
+        """
         data = self._behavior_stimulus_file()
         timestamps = self.get_stimulus_timestamps()
         # Get licks from pickle file (need to add an offset to align with
         # the trial_log time stream)
         lick_frames = (data["items"]["behavior"]["lick_sensors"][0]
                        ["lick_events"])
         lick_times = timestamps[lick_frames]
-        return pd.DataFrame({"time": lick_times})
+        return pd.DataFrame({"time": lick_times, "frame": lick_frames})
```

Comment:
Do this new `lick_frames` column need to be saved to NWB file?

Reply:
I don't know. It's required internally because the scientists want `get_trials` to assign lick times based on the timestamps of the frames (i.e. if a lick occurred in frame 1234, then the time of that lick is the timestamp of frame 1234, regardless of what the trial_log says the lick time should be).

I guess that means that, after running `get_trials`, `lick_times` and `lick_frames` are redundant.

Probably this is a question for Marina and company.

Reply:
@matchings Thoughts on if the new `frame` column for licks should be saved to NWB?

Reply:
`lick_times `should be derived from `lick_frames ` and either the ophys sync times, or the behavior only timebase based on intervalms, so yes they are essentially redundant. i dont think that `lick_frames ` needs to be in the NWB file. tagging @dougollerenshaw in case he wants to object, but for now i think its ok to leave them out.